pkg/webhook/pod/pods.go

```diff
@@ -78,7 +82,10 @@ func (pc *PodAdmissionControl) AdmitPods(ar v1beta1.AdmissionReview) *v1beta1.Ad
 	name := ar.Request.Name
 	namespace := ar.Request.Namespace
 	operation := ar.Request.Operation
-	klog.Infof("receive admission to %s pod[%s/%s]", operation, namespace, name)
+	if stsControllerServiceAccounts != ar.Request.UserInfo.Username {
```

Comment:
Make the skipping service account configurable

Reply:
fix